app/models/claim_checking_tasks.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# This models the tasks that need to be performed on a claim as part of the
+# claim checking process.
+class ClaimCheckingTasks
+  TASK_NAMES = %w[qualifications employment].freeze
+
+  attr_reader :claim
+
+  def initialize(claim)
+    @claim = claim
+  end
+
+  # Returns an Array of all the tasks that need to be performed on the claim
+  # during claim checking before it should be approved.
+  def applicable_tasks
```

Comment:
Reading the documentation and method name, I'd expect this method to return an array of `Task` instances. Perhaps we can be more precise with our words and distinguish between a task (`Task` instance) and a task name?

This is a point that applies more broadly across this PR but just going to mention it in one place until hearing your thoughts.